src/Wrapper.ts

```diff
@@ -62,22 +62,22 @@ export interface WrapperState<V> {
   isPristine: boolean;
   isRequired: boolean;
   isValid: boolean;
-  pristineValue: any;
-  validationError: any[];
+  pristineValue: V;
+  validationError: string[];
   value: V;
 }
 
 export interface InjectedProps<V> {
-  errorMessage: any;
-  errorMessages: any;
+  errorMessage: string;
+  errorMessages: string[];
```

Comment:
We haven't explicitly said that `errorMessage`(`s`) should be strings until now? Perhaps `ReactNode` would be  better – (I think that encompasses `string`)? This would prevent breakage if users were passing something else renderable like a React element/component as an error message?

Reply:
Interesting observation! I don't see why not. Good catch. I'll add to a PR I'm doing now.

I really appreciate your reviews. It doesn't seem like there's enough activity on formsy-react to get all my PRs reviewed but I find any comments you can find time to make very very helpful.